index.html

```diff
@@ -0,0 +1,105 @@
+          <li><a class="menu-item" href="#footer-cont">Copyright</a></li>
+          <li><a class="nav-btn" href="#campaing">Campaign</a></li>
+        </ul>
+      </div>      
+    </nav>
+    <div class="hero-item-cont">
+      <hgroup class="main-title">
+        <p class="tagline">"Hello World!"</p>
+        <h1 class="title">PERUVIAN SOFTWARE CONFERENCE 2023</h1>
+      </hgroup>
+      <p class="subheading">The Peruvian Software Conference (formerly Peruvian Science
+          and Information Conference) is a research conference held in Paracas, Perú 
+          since 2013. The annual conference series included keynotes, special sessions, 
+          poster presentations, tutorials, workshops and papers.</p>
+      <p class="conf-date">2023.10.13(FRI) - 15(SUN)</p>
+      <p class="conf-location">@ Hotel Paracas Paradise, Conference Room 5, Paracas, Perú.</p>
+    </div>    
+  </header>
+  <main id="main-cont">
+    <section id="prog-cont" class="prog-cont">
+      <h2 class="sec-title sec-white-title">Main Program</h2>
+      <div class="prog-card-cont flex-cont">
+        <div class="prog-card">
+          <img src="images/lecture.png" alt="">
+          <p class="prog-card-title">Lectures</p>
+          <p class="prog-card-desc">Listen to speakers from various countries about trends in software industry designed to inspire and inform.</p>
+        </div>
+        <div class="prog-card">
+          <img src="images/exhibition.png" alt="">
+          <p class="prog-card-title">PS Exhibition</p>
+          <p class="prog-card-desc">Insightful, unique and novel approaches to some of the world’s toughest problems are discussed.</p>
+        </div>
+        <div class="prog-card">
+          <img src="images/forum.png" alt="">
+          <p class="prog-card-title">Forum</p>
+          <p class="prog-card-desc">Supplies a free and open atmosphere where attendees talk and engage in creative ways.</p>
+        </div>
+        <div class="prog-card">
+          <img src="images/workshop.png" alt="">
+          <p class="prog-card-title">Workshop</p>
+          <p class="prog-card-desc">Our Software Workshop is one facility that brings together software researchers and engineers to transform basic research into powerful, standardized software systems.</p>
+        </div>
+        <div class="prog-card">
+          <img src="images/ignite.png" alt="">
+          <p class="prog-card-title">PS Ignite</p>
+          <p class="prog-card-desc">It allows users to find events, communicate with people attending events such as conferences, trade fairs and other events. It allows users to search for people with similar interests.</p>
+        </div>
+      </div>
+      
+      <button id="join-btn" class="join-btn" type="button">Join PS Conference 2023</button>
+      <a class="hidden" href="">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="partner-cont" class="partner-cont">
```

Comment:
- [ ] Good job 👏 However, from the design guidelines there is a difference between your design and the design given. Please look at the pictures below 👇 
**Given Design**
![image](https://user-images.githubusercontent.com/90222110/204053668-65ce748a-5be3-4335-be36-423d53be4ca3.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/204053726-0b57cf43-3882-49cb-8428-edd6fd742787.png)

But you should keep in mind that you will maintain the design for the mobile screen 😎